```
======================================================================
FAIL: test_user_similar (test.test_sysconfig.TestSysConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/churchyard/Dokumenty/RedHat/cpython/Lib/test/test_sysconfig.py", line 309, in test_user_similar
    self.assertEqual(user_path, expected)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: '/home/churchyard/.local/lib/python3.11/site-packages' != '/home/churchyard/.local/usr/lib64/python3.11/site-packages'
- /home/churchyard/.local/lib/python3.11/site-packages
+ /home/churchyard/.local/usr/lib64/python3.11/site-packages
?                         ++++   ++


----------------------------------------------------------------------
Ran 23 tests in 0.046s

FAILED (failures=1, skipped=2)
```

--- a/Lib/test/pythoninfo.py
+++ b/Lib/test/pythoninfo.py
@@ -96,6 +96,7 @@ def collect_sys(info_add):
         'maxunicode',
         'path',
         'platform',
+        'platlibdir',
         'prefix',
         'thread_info',
         'version',

